<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>new_image</class>
 <widget class="QDialog" name="new_image">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>379</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="1">
    <widget class="QTextEdit" name="heightInput">
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>30</height>
      </size>
     </property>
     <property name="placeholderText">
      <string>Height</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="measurements">
     <property name="title">
      <string>Measurements</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="radioButton_2">
        <property name="text">
         <string>Pixels</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton">
        <property name="text">
         <string>Centimeters</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_3">
        <property name="text">
         <string>Inches</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>New Image</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>Return</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QGroupBox" name="ColorScheme">
     <property name="title">
      <string>Color Scheme</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QRadioButton" name="radioButton_4">
        <property name="text">
         <string>RGB</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_6">
        <property name="text">
         <string>Inverse</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_5">
        <property name="text">
         <string>Greyscale</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QTextEdit" name="widthInput">
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>30</height>
      </size>
     </property>
     <property name="placeholderText">
      <string>Width</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Create</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QLabel" name="ErrorLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
